packages/ra-core/src/controller/list/useListController.storeKey.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
```

Comment:
for my information, why did you choose to put these tests in a separate file rather than in `useListController.spec.tsx`?